packages/astro/src/core/build/static-build.ts

```diff
@@ -13,7 +13,7 @@ import {
 } from '../../core/build/internal.js';
 import { emptyDir, removeEmptyDirs } from '../../core/fs/index.js';
 import { appendForwardSlash, prependForwardSlash } from '../../core/path.js';
-import { isModeServerWithNoAdapter } from '../../core/util.js';
+import { isModeServerWithNoAdapter, viteID } from '../../core/util.js';
```

Comment:
Looks like we can revert the changes in this PR as the added imports are unused.

There's also a few places that have unused imports. Running `pnpm format:ci` fixes it (a bit slow), but it's not a big deal if we don't fix it as CI will auto-format and remove it later. But it would help keep the git history clean.